docs/authorization.md

```diff
@@ -0,0 +1,171 @@
+# Authorization / Permission
+
+*WIP: The authorization is still not implemented in Perses*
+
+Perses will use a Role-based access control (RBAC) for regulating access to resources based on the role of the user.
+The RBAC API is based on four kinds of resource: GlobalRole, Role, GlobalRoleBinding and RoleBinding.
+Perses RBAC implementation is highly inspired by K8S RBAC implementation.
+
+
+## Role and GlobalRole
+
+An RBAC *Role* or *GlobalRole* contains a set of permissions (rules). Permissions are purely additive (there are no "deny" permissions).
+
+A *Role* set permission within a particular project, when you create a *Role* you need to specify the project it belongs in.
+*GlobalRole*, by contrast, is not limited to a project scope.
+
+
+### Role example
+
+Here is an example of Role in "MySuperProject" project that can be used to grant edit access to dashboards:
+```yaml
+kind: Role
+metadata:
+  name: dashboard-editor
+  project: MySuperProject
+spec:
+  permissions:
+    - action: edit
+      scopes: ["Dashboard"]
+```
+
+
+### Global Role example
+
+A *GlobalRole* can be used to grant the same permissions as a Role. However, because *GlobalRole* are global, you can also use them to grant access to:
```

Comment:
```suggestion
A `GlobalRole` can be used to grant the same permissions as a Role. However, because `GlobalRole` are global, you can also use them to grant access to:
```